pkgs/applications/networking/p2p/lokinet/default.nix

```
@@ -0,0 +1,69 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, curl
+, libevent
+, libsodium
+, libuv
+, pkgconfig
+, sqlite
+, unbound
+, writeShellScriptBin
+, zeromq
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lokinet";
+  version = "0.9.9";
+  githash = "440b547d2c3e7384ec82ec21f0b1635e42bcda64";
+
+  src = fetchFromGitHub {
+    owner = "oxen-io";
+    repo = "lokinet";
+    rev = "v${version}";
+    sha256 = "AaGsRg9S9Cng9emI/mN09QSOIRbE+x3916clWAwLnRs=";
+    fetchSubmodules = true;
+  };
+
+  fakegit = writeShellScriptBin "git" "echo ${githash}";
+
+  nativeBuildInputs = [
+    cmake
+    curl
+    fakegit
+    pkgconfig
+  ];
+
+  buildInputs = [
+    libevent
+    libuv
+    libsodium
+    sqlite
+    unbound
+    zeromq
+  ];
+
+  cmakeFlags = [
+    "-DWITH_BOOTSTRAP=OFF" # we provide bootstrap files manually
+    "-DWITH_LTO=OFF"
+    "-DWITH_SETCAP=OFF"
+    "-DWITH_SHARED_LIBS=OFF"
+    "-DWITH_SYSTEMD=OFF"
+  ];
+
+  # copy bootstrap files
+  # see https://github.com/oxen-io/lokinet/issues/1765#issuecomment-938208774
+  postInstall = ''
+    mkdir -p $out/share/testnet
+    cp $src/contrib/bootstrap/mainnet.signed $out/share/bootstrap.signed
+    cp $src/contrib/bootstrap/testnet.signed $out/share/testnet/bootstrap.signed
```

```suggestion
    cp contrib/bootstrap/mainnet.signed $out/share/bootstrap.signed
    cp contrib/bootstrap/testnet.signed $out/share/testnet/bootstrap.signed
```
Since we should be still in that directory.

---

Then maybe a .. is missing. Both is actually fine.